storm-core/src/jvm/org/apache/storm/command/GetErrors.java

```diff
@@ -0,0 +1,59 @@
+package org.apache.storm.command;
+
+import org.apache.storm.generated.ErrorInfo;
+import org.apache.storm.generated.GetInfoOptions;
+import org.apache.storm.generated.Nimbus;
+import org.apache.storm.generated.NumErrorsChoice;
+import org.apache.storm.generated.TopologyInfo;
+import org.apache.storm.utils.NimbusClient;
+import org.apache.storm.utils.Utils;
+import org.json.simple.JSONValue;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class GetErrors {
+    public static void main(String[] args) throws Exception {
+        final String name = args[0];
```

Comment:
It would be nice to have a check on args length here and throw "Topology name not passed" exception in case args is empty.